doc/modules/reference.rst

```diff
@@ -84,6 +84,9 @@ uses.
    fetch_localizer_calculation_task
    fetch_miyawaki2008
    fetch_nyu_rest
+   fetch_surf_nki_enhanced
+   fetch_surf_fsaverage5
+   fetch_atlas_surf_destrieux
```

Comment:
You may have missed adding all "plotting" tools here. Like what you have done with fetchers.